diff --git a/node_modules/@docusaurus/mdx-loader/lib/remark/toc/search.js b/node_modules/@docusaurus/mdx-loader/lib/remark/toc/search.js
index 24bff9a..1f48107 100644
--- a/node_modules/@docusaurus/mdx-loader/lib/remark/toc/search.js
+++ b/node_modules/@docusaurus/mdx-loader/lib/remark/toc/search.js
@@ -18,7 +18,7 @@ function search(node) {
     let currentDepth = 0;
     const visitor = (child, _index, parent) => {
         const value = mdast_util_to_string_1.default(child);
-        if (parent !== node || !value || child.depth > 3 || child.depth < 2) {
+        if (!value || child.depth > 3 || child.depth < 2) {
             return;
         }
         const entry = {
